packages/core/content-manager/admin/src/components/Widgets.tsx

```diff
@@ -0,0 +1,185 @@
+import { Widget, useTracking } from '@strapi/admin/strapi-admin';
+import { Box, IconButton, Status, Table, Tbody, Td, Tr, Typography } from '@strapi/design-system';
+import { Pencil } from '@strapi/icons';
+import { useIntl } from 'react-intl';
+import { Link, useNavigate } from 'react-router-dom';
+import { styled } from 'styled-components';
+
+import { useGetRecentDocumentsQuery } from '../services/homepage';
+import { capitalise } from '../utils/strings';
+
+import { RelativeTime } from './RelativeTime';
+
+import type { RecentDocument } from '../../../shared/contracts/homepage';
+
+const CellTypography = styled(Typography).attrs({ maxWidth: '14.4rem', display: 'block' })`
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+`;
+
+interface DocumentStatusProps {
+  status: RecentDocument['status'];
+}
+
+const DocumentStatus = ({ status = 'draft' }: DocumentStatusProps) => {
+  const statusVariant =
+    status === 'draft' ? 'secondary' : status === 'published' ? 'success' : 'alternative';
+
+  const { formatMessage } = useIntl();
+
+  return (
+    <Status variant={statusVariant} size="XS">
+      <Typography tag="span" variant="omega" fontWeight="bold">
+        {formatMessage({
+          id: `content-manager.containers.List.${status}`,
+          defaultMessage: capitalise(status),
+        })}
+      </Typography>
+    </Status>
+  );
+};
+
+const RecentDocumentsTable = ({ documents }: { documents: RecentDocument[] }) => {
+  const { formatMessage } = useIntl();
+  const { trackUsage } = useTracking();
+  const navigate = useNavigate();
+
+  const getEditViewLink = (document: RecentDocument): string => {
+    // TODO: import the constants for this once the code is moved to the CM package
+    const isSingleType = document.kind === 'singleType';
```

Comment:
Are we talking about the COLLECTION_TYPES/SINGLE_TYPES constant? Because that returns kebab case. I'm cool with this as is but lmk.

```suggestion
    const isSingleType = document.kind === 'singleType';
```

Reply:
oh yes i probably didn't realize it wasn't the right casing